frappe/patches.txt

```diff
@@ -87,7 +88,6 @@ frappe.patches.v11_0.set_missing_creation_and_modified_value_for_user_permission
 frappe.patches.v11_0.set_default_letter_head_source
 frappe.patches.v12_0.set_primary_key_in_series
 execute:frappe.delete_doc("Page", "modules", ignore_missing=True)
-frappe.patches.v11_0.set_default_letter_head_source
```

Comment:
removed cause duplicate

Reply:
INI parser will automatically fail when duplicates are added (which is kinda inline with how `patch log` is implemented anyway.